provider/datagen/src/transform/cldr/displaynames/language.rs

```diff
@@ -96,7 +96,7 @@ mod tests {
     fn test_basic_lang_display_names() {
         let provider = crate::DatagenProvider::for_test();
 
-        let data: DataPayload<LanguageDisplayNamesV1Marker> = provider
+        let data: DataPayload<LanguageDisplayNameToValueV1Marker> = provider
```

Comment:
Issue: More places where you need to change back the displaynames structs (this should fail to compile, right?)